CONTRIBUTING.md

```diff
@@ -161,7 +161,7 @@ This tag is for boolean properties associated with the test.
 #### features
 **features**: [list]
 
-Some tests require the use of language features that are not directly described by the test file's location in the directory structure. These features should be specified with this tag. See the `features.txt` file for a complete list of available values.
+Some tests require the use of language features that are not directly described by the test file's location in the directory structure. These features should be specified with this tag. See the `features.txt` file for a complete list of available values. This tag is required for new tests written for new features, but contributions will not be "blocked" if the tag is missing from frontmatter. The committing maintainer is required to ensure that the tag is present and contains the correct feature names; this can be done in an follow up commit. 
```

Comment:
Which consumers of test262 use the features metadata? It says above that it's required, here it says it's not really required, and it is something that seems to come up fairly often in recent PRs I've looked at.

Reply:
V8 uses some of the feature flags to switch off tests or switch on flags, expect the test to pass or fail, require/disable strict mode, or use modules; e.g., see https://github.com/v8/v8/blob/master/test/test262/testcfg.py#L43 . However, other parts of the front matter are purely documentation.

Reply:
> Which consumers of test262 use the features metadata?

Presently V8, SpiderMonkey, ChakraCore, Prepack and Flow. Babylon, Babel and JSHint have "works in progress" that use `features`. Features will be used for categorizing/filtering the dashboard. 

> It says above that it's required, here it says it's not really required

This is a very new compromise. There has been some feedback that typing those things is too much of a contributor burden, so Test262 is committing to taking on the burden when a contributor is unwilling to contribute according to the needs of the project. There is an effort underway to automate the process of tagging features. 

> is something that seems to come up fairly often in recent PRs I've looked at.

Yep, usually this is just a matter of asking for the addition.